.travis.yml

```diff
@@ -261,11 +259,12 @@ before_install:
 install:
   - source scripts/install-ci-dependencies.sh
   - source scripts/setup-helics-ci-options.sh
+  - export CMAKE_COMPILER_LAUNCHER="-DCMAKE_C_COMPILER_LAUNCHER=ccache -DCMAKE_CXX_COMPILER_LAUNCHER=ccache"
   
 script:
   - mkdir -p build && cd build
-  - cmake .. ${JOB_OPTION_FLAGS} ${HELICS_DEPENDENCY_FLAGS} ${HELICS_OPTION_FLAGS} -DCMAKE_C_COMPILER_LAUNCHER=ccache -DCMAKE_CXX_COMPILER_LAUNCHER=ccache
-  - make ${MAKEFLAGS}
+  - cmake -G "${CMAKE_GENERATOR}" ${JOB_OPTION_FLAGS} ${HELICS_DEPENDENCY_FLAGS} ${HELICS_OPTION_FLAGS} ${CMAKE_COMPILER_LAUNCHER} ..
```

Comment:
Denis had a request to echo the full expanded cmake command in the build log so we could easily see which options were being used for cmake on the different tests.  The variable are useful but they make it difficult to see what we are doing sometimes.